tests/test_inlineresult.py

```diff
@@ -0,0 +1,442 @@
+#!/usr/bin/env python
+#
+# A library that provides a Python interface to the Telegram Bot API
+# Copyright (C) 2015-2016
+# Leandro Toledo de Souza <devs@python-telegram-bot.org>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see [http://www.gnu.org/licenses/].
+
+"""This module contains a object that represents Tests for Telegram
+InlineResults"""
+
+import os
+import unittest
+import sys
+sys.path.append('.')
+
+import telegram
+from tests.base import BaseTest
+
+
+class InlineQueryResultArticleTest(BaseTest, unittest.TestCase):
+    """This object represents Tests for Telegram InlineQueryResultArticle."""
+
+    def setUp(self):
+        self.id = 'id'
+        self.title = 'title'
+        self.message_text = 'message text'
+        self.parse_mode = 'HTML'
+        self.disable_web_page_preview = True
+        self.url = 'url'
+        self.hide_url = True
+        self.description = 'description'
+        self.thumb_url = 'thumb url'
+        self.thumb_height = 10
+        self.thumb_width = 15
+
+        self.json_dict = {
+            'type': 'article',
```

Comment:
consider using a variable for `'article'` (i.e.:
```
self.type = 'article'
...
self.json_dict = {
    'type': self.type,
...
```
then later on you can call `assertEqual` with `self.type`